### DISCUSSION TITLE
About internet and external use

### DISCUSSION THREAD

**Discussion by InFlames513** [HenestrosaDev/audiotext#15] [General]
Hello, does this program require an internet connection or does it run locally? Also, can I integrate this program as a module in my python code, I need a module that works faster than gtts.

**Comment by HenestrosaDev** [comment]
Hello!

1. The Whisper models run on your machine and have no ability to transfer data to anywhere by themselves, so you can use Audiotext without Internet connection.
2. I think you would be better off using [WhisperX](https://github.com/m-bain/whisperX) directly instead of this project.
